test/integration/integration_test.go

```diff
@@ -103,25 +104,42 @@ var _ = Describe("Integration", func() {
 			Expect(json.Unmarshal(session.Wait().Out.Contents(), &result)).To(Succeed())
 		})
 
-		It("gets MicroVm", func() {
-			cmd := command{action: "get", args: []string{"--id", *result.Microvm.Spec.Uid}}
-			session := executeCommand(cmd)
-			Eventually(session).Should(gexec.Exit(0))
+		Context("with no args", func() {
+			It("gets the MicroVM from the default name and namespace", func() {
+				cmd := command{action: "get"}
+				session := executeCommand(cmd)
+				Eventually(session).Should(gexec.Exit(0))
 
-			var getResult v1alpha1.GetMicroVMResponse
+				var list v1alpha1.ListMicroVMsResponse
+				Expect(json.Unmarshal(session.Wait().Out.Contents(), &list)).To(Succeed())
+				Expect(list.Microvm[0].Spec.Id).To(Equal(defaultName))
+				Expect(list.Microvm[0].Spec.Namespace).To(Equal(defaultNamespace))
 
-			Expect(json.Unmarshal(session.Wait().Out.Contents(), &getResult)).To(Succeed())
-			Expect(getResult.Microvm.Spec.Id).To(Equal(result.Microvm.Spec.Id))
-		})
+			})
 
-		It("gets the state of the MicroVm", func() {
-			cmd := command{action: "get", args: []string{"--id", *result.Microvm.Spec.Uid, "--state"}}
-			session := executeCommand(cmd)
-			Eventually(session).Should(gexec.Exit(0))
-			Eventually(session.Out, "30s").Should(gbytes.Say("CREATED"))
+			Context("when more than one MicroVM with default name and namespace exist", func() {
```

Comment:
can we move this outside of the above nested context? i get what you are doing, but the feature we are testing is what happens when any name/ns is duped, so it is not about being default
```suggestion
			Context("when more than one MicroVM with in the same name and namespace group exist", func() {
```